### PR TITLE
feat(examples): OP Stack bridge stats ExEx

### PR DESCRIPTION
**Pull request by shekhirin**
image

### CODE REVIEW

**Comment by gakonst** [comment]
So good

File: examples/exex/optimism/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        MANTA => Some("Manta"),
+        _ => None,
+    }
+}
+
+fn main() -> eyre::Result<()> {
+    reth::cli::Cli::parse_args().run(|builder, _| async move {
+        let handle = builder
+            .node(EthereumNode::default())
+            .install_exex("Optimism", move |ctx| futures::future::ok(OptimismExEx { ctx }))
```
Comment (mattsse) [inline examples/exex/optimism/src/main.rs:127]: can use async move {Ok()}

**Review by mattsse — CHANGES_REQUESTED** [review]
cosmetic nits

File: examples/exex/optimism/src/main.rs (hunk 1 above)
Comment (shekhirin) [reply examples/exex/optimism/src/main.rs:127]: https://github.com/rust-lang/rust/issues/62290

Comment (shekhirin) [reply examples/exex/optimism/src/main.rs]: but `move |ctx| async { Ok()) }` should work 😄

**Review by mattsse — APPROVED** [review]

File: examples/exex/optimism/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+impl<Node: FullNodeTypes> Future for OptimismExEx<Node> {
+    type Output = eyre::Result<()>;
+
+    fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
+        let this = self.get_mut();
+
+        // Process all new chain state notifications until there are no more
+        while let Poll::Ready(Some(notification)) = this.ctx.notifications.poll_recv(cx) {
```
[the remark this answers is not in the document]
Comment (shekhirin) [reply examples/exex/optimism/src/main.rs:95]: > nit can use ` while let Some(notification) = ready!(`

yes

> an return Ok(()) at end of fn

why? This future should never be resolved, no?

**Review by mattsse — APPROVED** [review]

**Review by onbjerg — APPROVED** [review]
gg

**Pull request merged**